modules/service-accounts-as-oauth-clients.adoc

```diff
@@ -22,7 +22,7 @@ When using a service account as an OAuth client:
 +
 [source,terminal]
 ----
-$ oc sa get-token <service_account_name>
+$ oc create token <service_account_name>
```

Comment:
@ibihim I'm trying to get rid of places that use the deprecated `oc sa` command. If we update to use `oc create token`, that will use bound tokens (1hr) instead of long-lived tokens. Do you know whether this makes sense to change here, or if there's something else we should do?

Per @sanchezl, if they need longer than 1 hr, they could use --duration to specify a longer time, or we could have them create a legacy token.

Reply:
We would need to create a bound sa token. This would mean, that as long as the thing it is bound to exists, it will be valid AFAIU.

1. We would need to create a secret first: `oc create secret generic <secret_binding_the_token>`
2. Create a token and bind it: `oc create token <service_account_name> --bound-object-kind Secret --bound-object-name <secret_binding_the_token>`.